packages/table/src/utils/getTestUtils/getTestUtils.tsx

```diff
@@ -0,0 +1,112 @@
+import { getByLgId, queryBySelector } from '@lg-tools/test-harnesses';
+
+import { LGIDS } from '../../constants';
+
+import { TestUtilsReturnType } from './getTestUtils.types';
+
+export const getTestUtils = (
+  lgId: string = LGIDS.root,
+): TestUtilsReturnType => {
+  /**
+   * Queries the DOM for the element using the `data-lgid` data attribute.
+   * Will throw if no element is found.
+   */
+  const element: HTMLTableElement = getByLgId!(lgId);
+
+  const getAllHeaders = (): Array<HTMLTableCellElement> => {
+    const allHeaders = element.querySelectorAll<HTMLTableCellElement>(
+      `[data-lgid=${LGIDS.header}]`,
+    );
+
+    if (!allHeaders.length)
+      throw new Error('Unable to find any <th> elements.');
+
+    return Array.from(allHeaders);
+  };
+
+  const getHeaderByIndex = (index: number) => {
+    const allHeader = getAllHeaders();
+    const header = allHeader[index];
+    if (!header) return null;
```

Comment:
should we log something here if no header is found at `index`?

Reply:
I believe returning null should be sufficient and this is noted in the README. However, I made a change to all test harnesses so that an error is thrown if `querySelectorAll` returns a length of 0.